### PR TITLE
Support multiple validators

### PR DESCRIPTION
**Pull request by marctorsoc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,10 @@ results, exceptions = runner.run()
 ### Validating the response
 We can provide custom validation logic to the Runner - to retry the request if the response
 does not meet our criteria - for example, if it does not conform to the schema we expect. This
-assumes that the API is non-deterministic.
+assumes that the API is non-deterministic. The `validators` arguments accepts:
+- `None`: when no validators are needed
+- a single function: to be used for all requests
+- a list of functions: to be used for each request separately. In such a case, they'll be combined with and AND operator in the given order, i.e. if one of them returns `False`, the rest of calls will be skipped.
```
Comment (tpietruszka) [inline README.md:89]:
```suggestion
- a list of functions: to be used for each request separately. In such a case, they'll be combined with an AND operator in the given order, i.e. if one of them returns `False`, the rest of calls will be skipped.
```

File: README.md [hunk 2]
```diff
@@ -94,7 +97,7 @@ def character_number_is_even(response):
 validating_runner = Runner(
     openai.ChatCompletion.create,
     resources,
-    validation_function=character_number_is_even,
+    validators=character_number_is_even,
```
Comment (tpietruszka) [inline README.md:100]: Nit on naming: plural suggests that multiple (or at least a list) are required. Is there any way to name it better? (I don't really know right now)

Also, technically we're breaking compatibility, so SemVer-wise it should be a major release 😅

File: tests/test_runner.py [hunk 3]
```diff
@@ -276,7 +276,7 @@ def validate(result: dict) -> bool:
     runner = Runner(
         random_client,
         resources=dummy_resources(num_requests=5),
-        validation_function=validate,
+        validators=validate,
```
Comment (tpietruszka) [inline tests/test_runner.py:279]: Could you add a test please?

**Review by tpietruszka — APPROVED** [review]
Thanks a lot, it's a good idea, sorry it took me ages to review.

Overall looks good to me, except:
- I wonder if naming could be better
- please bump the version

File: README.md (hunk 2 above)
Comment (marctorsoc) [reply README.md:100]: I propose moving to `validation`, not implying singular or plural. Ok, so let's make it a major release then 😅

File: tests/test_runner.py (hunk 3 above)
Comment (marctorsoc) [reply tests/test_runner.py:279]: done

**Comment by tpietruszka** [comment]
Looks great, thanks!

**Pull request merged**